apps/platforms/win/wsl/wsl.talon

```diff
@@ -1,4 +1,8 @@
-os: windows
--
-
-(wsl|weasel) reset path detection: user.wsl_reset_path_detection()
+os: windows
+title: /^WSL:/
```

Comment:
I'm a little leery of a global regex based on the windows title creating false positives, though it's probably very unlikely in practice. 

For clarity, is your thinking to assert the wsl tag here, or in an app-specific manner as done in ubuntu.talon?

Reply:
Yes, you're right. TBH, I did actually hit at least one false positive while working on this change...a web page whose title just happened to match.

My thinking was to put the context match here in the .talon file so it would be accessible to the user without having to dig into wsl.py.

I explored matching on the app name instead of window title, and hit some snags:

First, there's wsl.exe - what you get if you run wsl from the taskbar search.

Then, there are the various distro exe's under C:\Users\me\AppData\Local\Microsoft\WindowsApps. Matching on those names means you need to keep updating your .talon context match as distros as added/removed.

Then, there are the various terminal emulator exe's that may be used to host a distro session, e.g. windows terminal, conemu, etc. Another wrinkle is that these same executables may also host non-wsl sessions like powershell.

And you can run wsl from a command.exe or powershell session.

Rather than matching on the window title or exe name, I thought about rigging up a callback mechanism from inside the distro to a talon listener of some sort. As a way of setting the user.wsl tag, I mean. Perhaps just writing to a shared file, for example.

I decided using the window title was best because there are other reasons people often do that...namely, to keep track of your sessions. Docker is a sort of a meta-distro with many flavors. And, one might want to run custom distros or multiple copies of the same distro (e.g. dev, staging). The window title is a handy way of doing that and so it's familiar for many of us.

If we do use the window title, what is the best string/format to use?

In any case, I'm happy to follow your lead on this.